hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/ReplicationServer.java

```diff
@@ -174,6 +194,20 @@ public static final class ReplicationConfig {
     )
     private double outOfServiceFactor = OUTOFSERVICE_FACTOR_DEFAULT;
 
+    @Config(key = "server.workers", defaultValue = "10", description = "server workers.", tags = {
+        DATANODE, MANAGEMENT})
+    private int replicationServerWorkers;
+
+    public int getReplicationServerWorkers() {
+      return replicationServerWorkers;
+    }
+
+    public ReplicationConfig setReplicationServerWorkers(
+        int replicationServerWorkers) {
+      this.replicationServerWorkers = replicationServerWorkers;
+      return this;
+    }
+
```

Comment:
I'm not sure we need a separate config property for this.  Maybe it's better to use `replicationMaxStreams`, which is currently only used in `ReplicationSupervisor`.

Reply:
@adoroszlai Thank you for your review. I have changed it. PTAL.